src/main/java/org/kumoricon/registration/print/formatter/badgeimage/BadgeImage.java

```diff
@@ -39,27 +40,66 @@ public class BadgeImage {
 
     void drawImage(Image image, Rectangle area) {
         if (image == null) return;
+        g2.addRenderingHints(new RenderingHints(RenderingHints.KEY_RENDERING, RenderingHints.VALUE_RENDER_QUALITY));
         g2.drawImage(image, area.x, area.y, area.width, area.height, null);
     }
 
-    void drawStretchedImage(Image image, Rectangle area) {
+    void drawStretchedImage(Image image, Rectangle boundingArea) {
         if (image == null) return;
-        double imageWidth = image.getWidth(null);
```

Comment:
revert to original

Reply:
Created issue https://github.com/kumoricon/registration/issues/161 and copied the original code to that issue, leaving it up to whoever resolves that issue what code they want to use here.